endpoints/openrtb2/amp_auction_test.go

```diff
@@ -1334,6 +1335,18 @@ func (m *mockAmpExchange) HoldAuction(ctx context.Context, auctionRequest exchan
 		}},
 		Ext: json.RawMessage(`{ "errors": {"openx":[ { "code": 1, "message": "The request exceeded the timeout allocated" } ] } }`),
 	}
+
+	if m.extPrebidTargetingTest {
+		response = &openrtb2.BidResponse{
```

Comment:
Wouldn't it be better to just modify the above response object rather that creating a second response object for this case? It would also make it clearer what the `m.extPrebidTargetingTest` is signaling as to the behavior of the mock.